assets/app/scripts/controllers/settings.js

```diff
@@ -68,24 +68,29 @@ angular.module('openshiftConsole')
     DataService.list("limitranges", $scope, function(limitRanges) {
       $scope.limitRanges = limitRanges.by("metadata.name");
       $scope.emptyMessageLimitRanges = "There are no resource limits set on this project.";
-      // Make sure max and min have the same sets of keys so we can actually create a table
-      // cleanly from a view.
+      // Convert to a sane format for a view to a build a table with rows per resource type
       angular.forEach($scope.limitRanges, function(limitRange, name){
         angular.forEach(limitRange.spec.limits, function(limit) {
-          limit.min = limit.min || {};
-          limit.max = limit.max || {};
-          limit["default"] = limit["default"] || {};
+          limit._byResource = {};
```

Comment:
Suggest making this map a scope variable rather than modifying limit since that object could be shared by other controllers.